Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -201,7 +203,7 @@ void Font::platformInit()
     CGFloat adjustment = shouldUseAdjustment(m_platformData.font()) ? ceil((ascent + descent) * kLineHeightAdjustment) : 0;
 
     lineGap = ceilf(lineGap);
-    lineSpacing = ceil(ascent) + adjustment + ceil(descent) + lineGap;
+    float lineSpacing = ceil(ascent) + adjustment + ceil(descent) + lineGap;
     ascent = ceilf(ascent + adjustment);
```

Comment:
Similar error here with `ceilf`.

Reply:
The mix of `float` and `double` here seems sloppy, is different between 32-bit and 64-bit because of `CGFloat`, slower to convert back and forth, but likely harmless. Since the data in WebCore proper is kept as `float` it’s not like we get a significant benefit from higher precision in the code here.